Mathlib/Topology/UniformSpace/Cauchy.lean

```diff
@@ -787,4 +787,29 @@ theorem secondCountable_of_separable [SeparableSpace α] : SecondCountableTopolo
     refine ⟨_, ⟨y, hys, k, rfl⟩, (hts k).subset hxy, fun z hz => ?_⟩
     exact hUV (ball_subset_of_comp_subset (hk hxy) hUU' (hk hz))
 
+section DiscreteUniformity
+
+open Filter
+
+/-- A Cauchy filter in a discrete uniform space is contained in a principal filter-/
```

Comment:
Here, it should suffice that the *topology* is discrete (whatever the uniform structure is).

Reply:
(And even a bit less : it suffices that all points be open)